pkg/operator/config.go

```diff
@@ -107,6 +109,7 @@ func NewConfig() *Config {
 		ClusterAutoscalerReplicas:      DefaultClusterAutoscalerReplicas,
 		ClusterAutoscalerCloudProvider: DefaultClusterAutoscalerCloudProvider,
 		ClusterAutoscalerVerbosity:     DefaultClusterAutoscalerVerbosity,
+		ReleaseVersion:                 DefaultReleaseVersion,
```

Comment:
We should never report this value - so if you want this sort of marker value you need to fail before you set it.  It would be illegal to set a non-semantic version into the operator.